rollup.js

```diff
@@ -46,18 +47,18 @@ module.exports = ({ name, jsBundle = 'bundle.js', cssBundle = 'bundle.css', inst
         const relativeOutputDirStart = outputDir.indexOf(OUTPUT_DIR_ROOT);
 
         if (relativeOutputDirStart === -1) {
-            this.error({ message: `outputDir must contain '${OUTPUT_DIR_ROOT}'` });
+            this.error({ message: `outputDir must contain '${OUTPUT_DIR_ROOT}', was: ${outputDir}` });
         }
 
         const relativeOutputDir = outputDir.substring(relativeOutputDirStart);
```

Comment:
The `relativeOutputDir` name was and is very wonky. As it doesn't contain a relative but an absolute path (i.e. it starts with a `/`). However, I decided to not touch naming and really only fix the issue I was having 😀.